appdaemon/DOCS.md

```diff
@@ -32,6 +32,8 @@ this isn't needed.
 
 **Note**: _Remember to restart the add-on when the configuration is changed._
 
+:information_source: In order to edit the configuration using the File Editor add-on, you will need to **disable** the File Editor configuration option "Enforce Basepath"
```

Comment:
This is not related to this add-on. Sorry, but I don't want to add a flood of tips for other add-ons here. For example, I don't prefer the file editor myself at all, yet, I'm not going to add documentation all file access add-ons out there.

Nevertheless, thanks for being willing to contribute 👍 

../Frenck